usagov/vote-gov
# Vote 787 -> excluded links test update

link to ticket -> https://bixal-projects.atlassian.net/browse/VOTE-787

##Purpose 
This Pr will move the excluded links test into its own folder.  With the issues of running it headless that some of the state links are giving that are out of our control it will be best to run this manually each sprint to ensure that the links are still working as expected. 

##To Test 

- pull down branch 
- run `npm run start`
- spilt terminal 
- run `npm run cy:open` and selected that excluded links test and ensure they are passing 
- also very that excluded links test are in their own folder and do not run when running the command `npm run cy:proofer`
